packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -18,6 +18,15 @@ interface InputPhoneNumberProps
   readonly value: string;
   readonly onChange: (value: string) => void;
 
+  /**
+   * A pattern to specify the format to display the phone number in.
+   * For example if you want to display the format for Denmark
+   * https://en.wikipedia.org/wiki/National_conventions_for_writing_telephone_numbers#Denmark
```

Comment:
ah, so close!

the url pushed the args table to its limit 😆 
<img width="1576" alt="image" src="https://user-images.githubusercontent.com/15986172/236558276-8491e5a5-e15b-4d70-b430-dc5fc7d4b73a.png">

You can markdown this btw
```suggestion
   * For example if you want to display the format for [Denmark](https://en.wikipedia.org/wiki/National_conventions_for_writing_telephone_numbers#Denmark)
```
should keep it nice and tidy 
![image](https://user-images.githubusercontent.com/15986172/236558443-fe90274d-3108-43a6-acde-7993dc0ca79b.png)